nemo/collections/asr/metrics/wer.py

```diff
@@ -148,12 +148,16 @@ def ctc_decoder_predictions_tensor(
         """
         hypotheses = []
         # Drop predictions to CPU
-        predictions = move_dimension_to_the_front(predictions, self.batch_dim_index)
-        prediction_cpu_tensor = predictions.long().cpu()
+        if isinstance(predictions, torch.Tensor):
```

Comment:
Add comment what can other value be. Also you need to update `wer_bpe.py`, `rnnt_wer.py` and `rnnt_wer_bpe.py` to have same behaviour.

Reply:
Also, add a unittest for this in the asr metrics tests file.